Library/Formula/nspr.rb

```diff
@@ -32,12 +32,12 @@ def install
         --enable-strip
         --enable-pthreads
         --enable-ipv6
-        --enable-macos-target=#{MacOS.version}
       ]
+      args << "--enable-macos-target=#{MacOS.version}" if OS.mac?
       args << "--enable-64bit" if MacOS.prefer_64_bit?
       system "./configure", *args
       # Remove the broken (for anyone but Firefox) install_name
-      inreplace "config/autoconf.mk", "-install_name @executable_path/$@ ", "-install_name #{lib}/$@ "
+      inreplace "config/autoconf.mk", "-install_name @executable_path/$@ ", "-install_name #{lib}/$@ " if OS.mac?
```

Comment:
Yes. The build fails without it.

I didn't copy down the precise error message, but if you'd like I can uninstall and reinstall without it to show you what it is.